examples/tests/invalid/map_entries.test.w

```diff
@@ -0,0 +1,7 @@
+let map = { "1" => "foo", "2" => "bar" };
+            ^ Expected type to be "str", but got "num" instead
+
+// check typesafety
+let mapUnsafe = { "foo" => 1, "bar" => 2 };
+let someStr: str = mapUnsafe.entries().at(0).value;
+                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Expected type to be "str", but got "num" instead
```

Comment:
The first line shouldn't have an error, and the error annotation needs to be in a comment

```suggestion
let map = { "1" => "foo", "2" => "bar" };

let mapUnsafe = { "foo" => 1, "bar" => 2 };
let someStr: str = mapUnsafe.entries().at(0).value;
//                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Expected type to be "str", but got "num" instead
```

Reply:
Sorry for the first comment; I accepted a suggestion that contained that change, and I was on my mobile so I could not read it well 👯